arrows.lic

```diff
@@ -0,0 +1,88 @@
+custom_require.call(%w[common common-crafting])
+
+class Arrows
+  include DRC
+  include DRCC
+
+  def initialize
+    @settings = get_settings
+    @bag = @settings.crafting_container
+    @bag_items = @settings.crafting_items_in_container
+    @belt = @settings.engineering_belt
+
+    arg_definitions = [
+      [
+        { name: 'type', options: %w[couger-claw boar-tusk sabertooth angiswaerd hele'la basilisk elsralael soot-stained ice-adder jagged-horn drake-fang], variable: true, descriptions: "Which arrow type to create?" },
+        { name: 'lumber', regex: /\w+/i, variable: true, description: 'Type of lumber used.' },
+        { name: 'tools', options: %w[true false], variable: true, description: 'Get tools from clerk?'}
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    @type = args.type
+    @lumber = args.lumber || 'maple'
```

Comment:
❓ This arg isn't optional, can it ever be `nil`?

Reply:
Good question. I will test more.

Reply:
Removed or logic.

Reply:
There is still or logic in this version? Git push ?